test/unit/client/test_client.py

```diff
@@ -1464,3 +1472,111 @@ def test_device_pinner(mock_client):
 
     assert isinstance(pin, DevicePinner)
     assert pin.object_id == pin_id
+
+
+def test_create_zip(mock_client, mock_box_session):
```

Comment:
Do you need this test if `test_download_zip` calls `create_zip`?

Reply:
Don't need it, but just another test because it is a separate method.